src/types.rs

```diff
@@ -262,6 +262,27 @@ impl<'prev, 'now> SubmitArgs<'prev, 'now> {
         }
     }
 
+    /// Sets a timeout in microseconds to start waiting for a minimum of a single completion.
+    ///
+    /// Once the timeout expires, the kernel will return when a single completion has been received
+    /// instead of waiting for the minimum amount of completions specified by the `want` parameter
+    /// in the call to [`Submitter::submit_and_wait`](crate::Submitter::submit_and_wait) or
+    /// [`Submitter::submit_with_args`](crate::Submitter::submit_with_args).
+    ///
+    /// Available since 6.12. Use the
+    /// [`Parameters::is_feature_min_timeout`](crate::Parameters::is_feature_min_timeout) method to
+    /// check for availability.
+    #[inline]
+    pub fn min_wait_usec<'new>(mut self, min_wait_usec: u32) -> SubmitArgs<'now, 'new> {
```

Comment:
`min_wait_usec` is not a reference, so there is no need to update its lifetime here.